pi_mqtt_gpio/modules/lm75.py

```diff
@@ -0,0 +1,41 @@
+from pi_mqtt_gpio.modules import GenericSensor
+
+
+"""REQUIREMENTS = ("smbus",)"""
+CONFIG_SCHEMA = {
+    "i2c_bus_num": {
+        "type": "integer",
+        "required": True,
+        "empty": False
+    },
+    "chip_addr": {
+        "type": "integer",
+        "required": True,
+        "empty": False
+    }
+}
+
+LM75_TEMP_REGISTER = 0
+
+
+class Sensor(GenericSensor):
+    """
+    Implementation of GPIO class for the PCF8574 IO expander chip.
```

Comment:
This docstring still references the PCF8574 IO module
```suggestion
    Implementation of Sensor class for the LM75 temperature sensor.
```